app/src/main/java/com/hammer/talkbbokki/MainActivity.kt

```diff
@@ -27,6 +39,22 @@ class MainActivity : ComponentActivity() {
                 }
             }
         }
+
+        dataObserver()
+    }
+
+    private fun dataObserver() {
+        lifecycleScope.launch {
+            lifecycle.repeatOnLifecycle(Lifecycle.State.STARTED) {
+                viewModel.topicList.collect {
+                    when(it) {
+                        is TopicListUiState.Success -> Toast.makeText(this@MainActivity, it.list.map { it.name }.toString(), Toast.LENGTH_SHORT).show()
+                        is TopicListUiState.Loading -> Toast.makeText(this@MainActivity, "Loading", Toast.LENGTH_SHORT).show()
+                        is TopicListUiState.Error -> Toast.makeText(this@MainActivity, "Error", Toast.LENGTH_SHORT).show()
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)